easybuild/toolchains/compiler/intel_compilers.py

```diff
@@ -59,5 +61,20 @@ def _set_compiler_vars(self):
     def set_variables(self):
         """Set the variables."""
 
+        if self.options.get('oneapi', False):
+            self.COMPILER_CXX = 'icpx'
+            self.COMPILER_CC = 'icx'
+            self.COMPILER_F77 = 'ifx'
+            self.COMPILER_F90 = 'ifx'
+            self.COMPILER_FC = 'ifx'
+            # fp-model source is not supported by icx but is equivalent to precise
+            self.options.options_map['defaultprec'] = ['ftz', 'fp-speculation=safe', 'fp-model precise']
```

Comment:
`ftz` doens't work for me in `ipx` and `ipcx`

Reply:
@bartoldeman

Reply:
ftz works in icx 2022 but not in 2021 indeed. So we'll need a version check.

Reply:
@bartoldeman Please open an issue on this ASAP, so we don't forget to fix it. Or, if it's an easy fix, a PR, then we can still squeeze in the fix for the EasyBuild v4.6.0 release we'll push out this week...